core/storage/collection/gae_models.py

```diff
@@ -286,10 +309,14 @@ def _trusted_commit(
         collection_commit_log.update_timestamps()
         collection_commit_log.put()
 
+    # We have ignored [override] here because the signature of this method
```

Comment:
I don't see why you need this. Doesn't it match VersionedModel?

Reply:
Yes it matches with VersionedModel, but not with BaseModel(super of VersionedModel).
Error: ` Signature of "delete_multi" incompatible with supertype "BaseModel"  [override]`

Reply:
Oh I see. So even if this is mediated through an intermediate class (VersionedModel) the error still shows up at the child class level? If so, that's annoying, but I guess it is what it is.

Reply:
Yes. And we don't have anything to fix this!